blueprints/ember-timetree/index.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
   },
 
   afterInstall: function() {
-    var json = require('../../../../bower.json');
-    return this.addBowerPackageToProject('d3', json.version);
+    return this.addBowerPackageToProject('d3');
```

Comment:
That makes sense. I see what it was trying to do before. I got the import error trying to use the addon, so I blindly copied from the ember-cli docs. Will give this a spin.